assets/src/dashboard/app/views/myStories/content/storiesView/index.js

```diff
@@ -267,10 +267,23 @@ function StoriesView({
                   setFocusedStory({ id: activeStory.id });
                   setActiveDialog('');
                 }}
+                aria-label={sprintf(
+                  /* translators: %s is story title */
+                  __('Cancel deleting story "%s"', 'web-stories'),
+                  activeStory.title
+                )}
               >
                 {__('Cancel', 'web-stories')}
               </Button>
-              <Button type={BUTTON_TYPES.DEFAULT} onClick={handleOnDeleteStory}>
+              <Button
+                type={BUTTON_TYPES.DEFAULT}
+                onClick={handleOnDeleteStory}
+                aria-label={sprintf(
+                  /* translators: %s is story title */
```

Comment:
```suggestion
                  /* translators: %s: story title */
```

Reply:
@BrittanyIRL You resolved these without change or comment. Was that on purpose? :)

Reply:
oh weird, i swear i committed it and then resolved. i'll go make the change manually. sorry about that!

Reply:
https://github.com/google/web-stories-wp/pull/4642/commits/055fe846a9f737eb1a165edbc99fba617cb3df19 

all better